docs/playbook-reference/actions/scans.rst

```diff
@@ -0,0 +1,48 @@
+Scans
+########
+
+Robusta includes built-in actions to scan and get insights on Kubernetes clusters.
+
+These actions can be triggered automatically on a schedule, manually using the Robusta Platform and :ref:`manually using the robusta cli <Manual Triggers>`.
```

Comment:
These actions can be triggered:

* Automatically, on a schedule
* On demand, via the Robusta UI
* On demand, via cli command